apps/token-e2e/src/integration/flow/staking-flow.cy.js

```diff
@@ -435,7 +431,7 @@ context.skip('Staking Tab - with eth and vega wallets connected', function () {
 
       cy.get(vegaWalletStakedBalances, txTimeout)
         .should('contain', 2.0, txTimeout)
-        .and('contain', this.validatorName);
+        .and('contain', '');
```

Comment:
Question (non-blocking): why check for an empty string here?

Reply:
Nice spot. Corrected!